### ISSUE TITLE
How to make the model predict values?

### ISSUE THREAD

**Issue by KatameRonin** [KurochkinAlexey/AntisymmetricRNN#1]
I am trying to use the basic AntisymmetricRNN model to be able to make time series predictions. I am trying to train my model on sine and cos values. I am trying to train my model on 2*sin() and 2*cos() and try to predict 1.7*sin() when the window contains 1.7*cos(). I added a function before it that converts the data into a sliding window format for time series forecasting. 

So I changed the model to have 1 output class instead. Rest of the method is still the same. Also since i'm doing regression instead of classification, i have changed both the optimizer and the loss functions to Adam and MSELoss() respectively. However, as given in the code,  outs gives me the probability of the classes for a particular sample.  
So I understand according to the problem statement of the paper, this is what is desired, probability and the predicted class but if i want to predict values using it , what is it that i should do to adjust to my problem?
